### PR TITLE
address #133

### PR DESCRIPTION
**Pull request by amnona**
remove 2 tmp files
add the all.seqs.fa to doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ The input to Deblur workflow is a directory of FASTA or FASTQ files (1 per sampl
 
 - all.biom : contains all Deblurred reads. This file represents the union of the "reference-hit.biom" and "reference-non-hit.biom" tables.
 
+- all.seqs.fa : a fasta file containing all the sequences in all.biom
+
```
Comment (wasade) [inline README.md]: Please also describe the contents of `all.seqs.fa.sortmerna.blast` and `all.seqs.fa.no_artifacts`

File: scripts/deblur [hunk 2]
```diff
@@ -649,12 +649,14 @@ def workflow(seqs_fp, output_dir, pos_ref_fp, pos_ref_db_fp,
     create_otu_table(output_fp, samples,
                      outputfasta_fp=outputfasta_fp, minreads=min_reads)
 
-    # if negative mode, also create a biom table with
+    # also create a biom table with
     # only 16s sequences and with only non-16s sequences
```
Comment (wasade) [inline scripts/deblur]: not 16s specific

File: README.md (hunk 1 above)
Comment (amnona) [inline README.md]: all.seqs.fa.sortmerna.blast and all.seqs.fa.no_artifacts are gone :)
they are now added to the tmp_files and deleted at the end of the deblur pipeline unless --keep-tmp-files is selected
so no need to describe them :)

Comment (josenavas) [inline README.md]: Does this mean that the only output fasta that deblur will generate now is the all.seqs.fa? If so, I disagree with the change as the file is useful for subsequent analysis, like generate a tree

Comment (wasade) [inline README.md]: I agree. Keeping a file of the sequences represented in the BIOM table that will be used for downstream analysis, in the vast majority of cases, makes sense to produce.

Comment (antgonza) [inline README.md]: Agree.

**Comment by josenavas** [comment]
Just the comment about removing the sequence files - having the sequence file paired with the biom table is important for downstream analysis. Otherwise this looks good.

File: README.md (hunk 1 above)
Comment (wasade) [inline README.md]: So it turns out, neither of the existing two sequence files corresponded to `reference-hit.biom` (formerly the only-16s table). 

@amnona, what we need are:

- a FASTA corresponding to `reference-hit.biom`
- a FASTA corresponding to `reference-non-hit.biom`

File: deblur/workflow.py [hunk 3]
```diff
@@ -262,6 +262,25 @@ def filter_minreads_samples_from_table(table, minreads=1, inplace=True):
     return table
 
 
+def fasta_from_biom(table, fasta_file_name):
+    '''Save squences from a biom table to a fasta file
```
Comment (wasade) [inline deblur/workflow.py]: squences -> sequences

**Pull request merged**